modules/@apostrophecms/i18n/index.js

```diff
@@ -1251,6 +1257,53 @@ module.exports = {
             console.log(`Due to conflicts, kept ${kept} documents from ${keep}`);
           }
         }
+      },
+      'strip-slug-accents': {
+        usage: 'Remove Latin accent characters from all document slugs and attachment names. Usage: node app @apostrophecms/i18n:strip-slug-accents',
+        async task() {
+          let docChanged = 0;
+          let attachmentChanged = 0;
+
+          await self.apos.migration.eachDoc({}, 5, async doc => {
+            const slug = doc.slug;
+            const req = self.apos.task.getAdminReq({
+              locale: doc.aposLocale?.split(':')[0] || 'en'
+            });
+            if (!self.shouldStripAccents(req)) {
+              return;
+            }
+
+            doc.slug = self.apos.util.slugify(doc.slug, {
+              stripAccents: true,
+              allow: '/'
+            });
+            if (slug !== doc.slug) {
+              await self.apos.doc.update(req, doc, { permissins: false });
```

Comment:
There is a typo with `permissions`.

Updating using `self.apos.doc.update` means there are no redirects created. The old URLs give me 404s now.